website/docs/tutorials/react.md

```diff
@@ -663,9 +630,71 @@ export default function Inbox() {
 
 This will format the date using the conventional format for the active language.
 
+## Memoization Pitfall
```

Comment:
I have doubts about this section. Maybe it would be better to move it to the Lazy Translations article?